src/content/_index.en.md

```diff
@@ -19,7 +19,37 @@ Submariner is completely open source, and designed to be network plugin (CNI) ag
 * [subctl](./deployment/), a friendly deployment tool 
 * Support for interconnecting clusters with [overlapping CIDRs](./architecture/globalnet/)
 
+<style>
+.mygrid {  
+    display: grid;
+    grid-gap: 12px;  
+    grid-template-columns: repeat(auto-fit, minmax(180px, 1fr));
+    grid-template-rows: repeat(2, 100px);  
+    padding-bottom: 2em;
+}
+.mygrid h5 {
+    text-align: center;
+}
+
+</style>
+
+### Examples
+<div class="mygrid">
+  <div>
+    <div> {{< youtube fMhZRNn0fxQ >}}</div>
+    <h5> Connecting containers accross clusters</h5>
```

Comment:
"Connecting Pods and Services across Clusters"

Reply:
right I didn't check that :)

Reply:
```suggestion
    <h5> Connecting Pods and Services across Clusters</h5>
```